go/vt/vtctl/workflow/stream_migrator.go

```diff
@@ -201,8 +201,8 @@ func (sm *StreamMigrator) StopStreams(ctx context.Context) ([]string, error) {
 /* tablet streams */
 
 func (sm *StreamMigrator) readTabletStreams(ctx context.Context, ti *topo.TabletInfo, constraint string) ([]*VReplicationStream, error) {
-	query := fmt.Sprintf("select id, workflow, source, pos, workflow_type, workflow_sub_type, defer_secondary_keys from _vt.vreplication where db_name=%s and workflow != %s",
-		encodeString(ti.DbName()), encodeString(sm.ts.ReverseWorkflowName()))
+	query := fmt.Sprintf("select id, workflow, source, pos, workflow_type, workflow_sub_type, defer_secondary_keys from _vt.vreplication where db_name=%s and workflow=%s",
```

Comment:
We only migrate streams for reshard workflows. The above code is expected to select all appropriate streams (Materialize/CreateLookupVindex workflow types) that need to be moved to the new shards.